code/game/objects/items/handcuffs.dm

```diff
@@ -364,6 +364,37 @@
 	breakouttime = 70
 	knockdown = 20
 
+/obj/item/restraints/legcuffs/bola/nunchucks//curator traitor variant, basically the bola can also beat people up.
+	name = "nunchucks"
+	desc = "Pretty badass if you can use it right. You could probably use it to wrap around someone's legs like a bola as well, if they try running."
+	icon_state = "nunchuck"
+	item_state = "nunchuck"
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	hitsound = 'sound/weapons/nunchuck_hit.ogg'
+	slot_flags = ITEM_SLOT_BELT
+	force = 15
+	throwforce = 15
+	w_class = WEIGHT_CLASS_SMALL
+	attack_verb = list("attacked", "smacked", "bashed", "cracked")
+	block_chance = 10
+	max_integrity = 200
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+
+	breakouttime = 70
+	knockdown = 20
+
+/obj/item/restraints/legcuffs/bola/nunchucks/attack(mob/target, mob/living/user)
+	if(user.has_trait(TRAIT_CLUMSY) && target != user)
+		attack(user, user)
+		return
+	..()
+	if(prob(25))
+		user.emote("flip")
+	user.changeNext_move(CLICK_CD_RANGE)

```

Comment:
https://github.com/tgstation/tgstation/blob/d6b3dc5a3f7469cf31f9ca6a2f6c2e173c97196e/code/_onclick/item_attack.dm#L60-L65
This will happen even if the attack fails.